esrc/navtabs.php

```diff
@@ -2,10 +2,13 @@
 // start <ul>
 echo '<ul class="nav nav-tabs">';
 
+$navtabsSystem = isset($system) ? $system : "";
```

Comment:
I suggest to set here some value like "unknown system" over an empty string at least for '$navtabsActiveSAR'.